docs/docs/themes/getting-started.md

```diff
@@ -38,13 +59,8 @@ path: /hello-world
 I'm an example post!
 ```
 
-## 3. Scaffold Out the Home Page
-
-```md:title=src/pages/index.md
-# Home page!
-```
-
 ## Updating a Theme
 
-In order to update the theme to pull in the latest updates you can update the `gatsby-theme-blog` version in your `package.json`.
-You can do this by running the install again: `npm install --save gatsby-theme-NAME`.
+In order to update the theme to pull in the latest updates you can update the `gatsby-theme-blog` version in your site's `package.json`.
+
+You can do this by running the install of the theme package again: `npm install --save gatsby-theme-blog`.
```

Comment:
I think this doc would benefit from a recommendation of next steps from a content perspective. What would someone do next with what they have set up?